src/dotnet/ToolPackage/ToolPackageInstaller.cs

```diff
@@ -161,7 +161,17 @@ private string JoinSourceAndOfflineCache(string[] additionalFeeds)
             var feeds = new List<string>();
             if (additionalFeeds != null)
             {
-                feeds.AddRange(additionalFeeds);
+                foreach (var feed in additionalFeeds)
+                {
+                    if (Uri.IsWellFormedUriString(feed, UriKind.Absolute))
```

Comment:
Assuming URLs are valid feeds?

Reply:
NuGet supports both URIs and file paths, yes.

Reply:
Thanks- presuming perf doesn't matter here. If it does, you could use `Path.IsFullyQualified()` to find items that are for sure not going to change based on the CWD. You'd still have to check Uri for false as they aren't fully qualified (from the filesystem perspective).

Reply:
Thanks for the feedback.  We likely won't have more than a few of these `--source-feed` options passed on the command line as users would probably opt for an actual NuGet.config file in that case.  Do you think it'd be worth adding a `Path.IsPathFullyQualified` check to avoid a few `Path.GetFullPath` calls?